lib/topological_inventory/openshift/operations/worker.rb

```diff
@@ -36,37 +35,35 @@ def stop
 
         private
 
-        attr_accessor :messaging_client_opts, :client, :api_client, :sleep_poll, :poll_timeout
+        attr_accessor :messaging_client_opts, :client, :sleep_poll, :poll_timeout
 
         def process_message(client, msg)
           logger.info("Processing #{msg.message} with msg: #{msg.payload}")
           # TODO: Move to separate module later when more message types are expected aside from just ordering
-          order_service(client, msg)
+          order_service(msg.payload)
+          client.ack(msg.ack_ref)
```

Comment:
@agrare if the processing is done via thread (so the wait it not blocking) the manual ack is not needed, we should remove it

Reply:
@Ladas subscribe_messages only auto acks the last message of the whole batch so if we successfully processed half then failed it would retry the entire batch.  This is why we kept the manual ack.

Reply:
ok